pages/rounds.js

```diff
@@ -1,26 +1,30 @@
 import moment from 'moment-timezone';
 import React, { Fragment } from 'react';
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import sumBy from 'lodash/sumBy';
+import orderBy from 'lodash/orderBy';
```

Comment:
happy to change so long as we're on the same page about the impact. The way you are doing it, all of `lodash` will end up in the bundle. It's my understanding that the syntax I had is the only way to get tree shaking to only bring in the used methods.

Reply:
<img width="577" alt="Screen Shot 2020-10-08 at 5 44 44 PM" src="https://user-images.githubusercontent.com/157415/95528817-03482d80-098e-11eb-9a55-e7ad3d7a28c1.png">

Reply:
Ah right, the `lodash` package is annoying like that. Can we use `lodash-es` instead?

Reply:
Hard to tell the effect of the import style upon further googling. I just converted it to the preferred style and I can check what's happening with the bundle analyzer. 

Side note, I think `lodash-es` isn't a thing you need anymore with newer versions of webpack. but idk.

Reply:
From various SO question/answers I've seen, `lodash` is not tree shakeable by webpack no matter what because it's not written using ES modules. There are some babel plugins that can be used to prevent importing the whole package.

https://stackoverflow.com/a/62860462
https://stackoverflow.com/a/50656694
https://stackoverflow.com/a/48828219